ansible/roles/test/tasks/crm/crm_test_vnet_bitmap.yml

```diff
@@ -0,0 +1,187 @@
+- block:
+
+    - name: Clear FDB table
+      command: sonic-clear fdb all
+
+    - name: Make sure CRM counters updated
+      pause: seconds={{ crm_update_time }}
+
+    - name: Get "crm_stats_ipv4_route" used and available counter value
+      command: redis-cli --raw -n 2 HMGET CRM:STATS crm_stats_ipv4_route_used crm_stats_ipv4_route_available
+      register: out
+    - set_fact:
+        crm_stats_ipv4_route_used: "{{ out.stdout_lines[0] }}"
+        crm_stats_ipv4_route_available: "{{ out.stdout_lines[1] }}"
+
+    - name: Get "crm_stats_ipv4_nexthop" used and available counter value
+      command: redis-cli --raw -n 2 HMGET CRM:STATS crm_stats_ipv4_nexthop_used crm_stats_ipv4_nexthop_available
+      register: out
+    - set_fact:
+        crm_stats_ipv4_nexthop_used: "{{ out.stdout_lines[0] }}"
+        crm_stats_ipv4_nexthop_available: "{{ out.stdout_lines[1] }}"
+
+    - name: Get "crm_stats_ipv4_neighbor" used and available counter value
+      command: redis-cli --raw -n 2 HMGET CRM:STATS crm_stats_ipv4_neighbor_used crm_stats_ipv4_neighbor_available
+      register: out
+    - set_fact:
+        crm_stats_ipv4_neighbor_used: "{{ out.stdout_lines[0] }}"
+        crm_stats_ipv4_neighbor_available: "{{ out.stdout_lines[1] }}"
+
+    - name: Get "crm_stats_fdb_entry" used and available counter value
+      command: redis-cli --raw -n 2 HMGET CRM:STATS crm_stats_fdb_entry_used crm_stats_fdb_entry_available
+      register: out
+    - set_fact:
+        crm_stats_fdb_entry_used: "{{ out.stdout_lines[0] }}"
+        crm_stats_fdb_entry_available: "{{ out.stdout_lines[1] }}"
+
+    - name: Apply VNET config
+      include: roles/test/tasks/vnet_vxlan.yml
+      vars:
+        config: True
+        cleanup: False
+
+    - set_fact: cfg_removed=False
```

Comment:
What is the purpose of this variable?

Reply:
It is used to check whether config is removed or not, so in cleanup section we can:
- perform cleanup if for example config is already applied and playbook failed before "remove" logic
- don't perform cleanup in case playbook passed which means that config was already removed as part of test logic